sdcm/nemesis.py

```diff
@@ -1898,6 +1898,18 @@ def repair_nodetool_repair(self, node=None, publish_event=True):
         with adaptive_timeout(Operations.REPAIR, node, timeout=HOUR_IN_SEC * 48):
             node.run_nodetool(sub_cmd="repair", publish_event=publish_event)
 
+    def run_repair_on_nodes(self, nodes: list, publish_event=True):
+        """
+        Execute a nodetool repair on the specified nodes, disregarding errors that may
+        arise from failed or unavailable nodes during the process.
+        """
+        for node in nodes:
+            try:
+                with adaptive_timeout(Operations.REPAIR, node, timeout=HOUR_IN_SEC * 48):
```

Comment:
48 hours timeout for a single node? Why that big value?

We have a test scenario with 60 DB nodes.
How long will it take on a setup with 60 nodes?

Reply:
I wonder who's the one that suggest to do those repairs... (surprise surprise it was @fruch)

I would recommend using manager repair, or in case of tablets `nodetool cluster repair` 

doing those serially on all nodes, would be very slow and efficient

also please cross check this with the core team owning repair, what they suggestion and if they approve this

on note, on tablets this won't be 48, it would fail after 1h, and raise critical after 3h
regardless 48h doesn't make no sense in this situation, now days we don't run test for more then 12, just a handful longer ones during release